### PR TITLE
docs: Update notification docs

### PR DESCRIPTION
**Pull request by raosan**
Implementing issue #141

### CODE REVIEW

File: docs/src/pages/quick-start.md [hunk 1]
```diff
@@ -7,14 +7,15 @@ title: Quick Start
...
-- [config.example.json](https://github.com/hyperjumptech/monika/blob/main/config.example.json)
+1. [SMTP Gmail](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.smtp-gmail.example.json)
+2. [Mailgun](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.mailgun.example.json)
+3. [Sendgrid](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.sendgrid.example.json)
+4. [Webhook](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.webhook.example.json)
+5. [Slack](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.slack.example.json)
+6. [Telegram](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.telegram.example.json)
+7. [Whatsapp](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.whatsapp.example.json)
+8. [Microsoft Teams](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.teams.example.json)
+9. [Defult](https://github.com/hyperjumptech/monika/blob/main/config.example.json)
```
Comment (haricnugraha) [inline docs/src/pages/quick-start.md:18]: It should be Default.

File: docs/src/pages/quick-start.md [hunk 2]
```diff
@@ -7,14 +7,15 @@ title: Quick Start
...
-- [config.whatsapp.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.whatsapp.example.json)
-- [config.teams.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.teams.example.json)
-- [config.example.json](https://github.com/hyperjumptech/monika/blob/main/config.example.json)
+1. [SMTP Gmail](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.smtp-gmail.example.json)
+2. [Mailgun](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.mailgun.example.json)
+3. [Sendgrid](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.sendgrid.example.json)
+4. [Webhook](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.webhook.example.json)
+5. [Slack](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.slack.example.json)
+6. [Telegram](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.telegram.example.json)
+7. [Whatsapp](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.whatsapp.example.json)
```
Comment (haricnugraha) [inline docs/src/pages/quick-start.md:16]: It should be WhatsApp

File: docs/src/pages/quick-start.md [hunk 3]
```diff
@@ -7,14 +7,15 @@ title: Quick Start
...
-- [config.mailgun.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.mailgun.example.json)
-- [config.sendgrid.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.sendgrid.example.json)
-- [config.webhook.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.webhook.example.json)
-- [config.slack.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.slack.example.json)
-- [config.whatsapp.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.whatsapp.example.json)
-- [config.teams.example.json](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.teams.example.json)
-- [config.example.json](https://github.com/hyperjumptech/monika/blob/main/config.example.json)
+1. [SMTP Gmail](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.smtp-gmail.example.json)
+2. [Mailgun](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.mailgun.example.json)
+3. [Sendgrid](https://github.com/hyperjumptech/monika/blob/main/config_sample/config.sendgrid.example.json)
```
Comment (haricnugraha) [inline docs/src/pages/quick-start.md:12]: It should be SendGrid.

File: docs/src/pages/quick-start.md (hunk 1 above)
Comment (raosan) [reply docs/src/pages/quick-start.md]: nice catch. fixed

**Review by haricnugraha — APPROVED** [review]

File: docs/src/pages/guides/notifications.md [hunk 4]
```diff
@@ -9,14 +9,15 @@ Monika will send notifications to you whenever [alerts](https://hyperjumptech.gi
...
-2. Webhook
-3. Mailgun
-4. Sendgrid
-5. Slack
-6. Whatsapp
-7. Microsoft Teams
-8. Discord
+1. [SMTP](https://hyperjumptech.github.io/monika/guides/notifications#smtp)
+2. [Mailgun](https://hyperjumptech.github.io/monika/guides/notifications#mailgun)
+3. [SendGrid](https://hyperjumptech.github.io/monika/guides/notifications#mailgun)
```
Comment (haricnugraha) [inline docs/src/pages/guides/notifications.md:14]: I missed this. Link should be https://hyperjumptech.github.io/monika/guides/notifications#sendgrid.

**Review by kevinhermawan — APPROVED** [review]

**Pull request merged**